contrib/utilities/indent

```diff
@@ -71,19 +71,27 @@ process_changed "source" ".*\.inst.in" format_inst
 process_changed "tests include source examples cmake/scripts contrib/python-bindings" \
   ".*\.(cc|h|inst.in|output.*|cmake)" fix_permissions
 
+process_changed "doc/news/*/*" "*" fix_permissions
```

Comment:
As reported in #16427, this captures the top-level directory for me as well, and then changes a *directory's* permissions to 644, which makes it inaccessible.